core/koin-core/build.gradle

```diff
@@ -71,6 +71,7 @@ kotlin {
             dependsOn commonMain
             dependencies {
                 implementation 'org.jetbrains.kotlin:kotlin-stdlib-js'
+                implementation "co.touchlab:stately-concurrency:1.2.2"
```

Comment:
I'm approving this PR because the entire code make senses. But I don't know if @kotzilla-io thinks in go on with add or be accoupled with other library.

It's not a techinical issue, it's just approach, ok?

Reply:
This library is already in use for the native platform. To be more specific, for the `co.touchlab.stately.concurrency.Lock`.
I just included the same version for the JS platform.
So no new dependencies were added.